### PR TITLE
Add issue templates, template links, and script to generate them

### LINKED ISSUE #15578: Add Issue template
Ran into this cool little feature while searching for something. Looks useful for prefilling in fenced code blocks and other niceties.

http://archetyped.com/know/guided-github-reports/

Opening this issue to get some feedback & to track progress.

### PR DESCRIPTION
**Pull request by jawshooah**
Closes #15578.

[Review on Reviewable](https://reviewable.io/reviews/caskroom/homebrew-cask/15758)

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Looks good.

File: doc/issue_templates/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+### Justification
+
+
+
+### Example use case
+
+
+
+# Cask request
```
Comment (Amorymeltzer) [inline doc/issue_templates/feature_request.md]: This line feels out of place?  I'm not sure what it's point is

Comment (vitorgalvao) [inline doc/issue_templates/feature_request.md]: Nice catch. Looks like this one is an error, further confirmed by looking at [the original proposals](https://github.com/caskroom/homebrew-cask/issues/15578#issuecomment-163490426).

Comment (jawshooah) [inline doc/issue_templates/feature_request.md]: Whoops, that was a goof :flushed:

Will fix.

**Comment by jawshooah** [comment]
I also added a script `update_issue_template_urls`, which you can use like so:

```
# make changes to templates
./developer/bin/update_issue_template_urls
git commit -am "Update issue templates"
```

File: doc/issue_templates/feature_request.md (hunk 1 above)
Comment (jawshooah) [inline doc/issue_templates/feature_request.md]: Fixed in 8127c20.

**Comment by jawshooah** [comment]
@Amorymeltzer any other comments before I merge?

**Comment by Amorymeltzer** [comment]
@jawshooah Just to confirm, in `doc/issue_templates/cask_request.md`, is your intent is to have those on separate lines or one line?  One line (the way it is currently, yay markdown) looks a little weird, but I imagine you intended for another newline between them

**Comment by jawshooah** [comment]
Huh, they're on separate lines in the preview. [Have a look](https://github.com/caskroom/homebrew-cask/issues/new?title=Cask%20request%3A&body=%23%23%23%20Cask%20details%0A%0A%28Please%20fill%20out%20as%20much%20as%20possible%29%0A%0A%2A%2AName%2A%2A%20-%0A%2A%2AHomepage%2A%2A%20-%0A%2A%2ALicense%2A%2A%20-%0A%2A%2ADownload%20URL%2A%2A%20-%0A%2A%2ADescription%2A%2A%20-%0A).

**Comment by jawshooah** [comment]
> the link itself loads just fine; it's just the doc renders odd.

Ah, I didn't realize. Should we just double-space them for consistency, then?

**Comment by jawshooah** [comment]
Went ahead and double-spaced the fields in `cask_request.md` before merging.

**Pull request merged**